### PR TITLE
KAFKA-3314: Add CDDL license to LICENSE and NOTICE file

### PR DESCRIPTION
**Pull request by junrao**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -200,3 +200,6 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+This distribution has a binary dependency on jersy, which is available under the CDDL
```
Comment (ijuma) [inline LICENSE]: Typo `jersy`

File: NOTICE [hunk 2]
```diff
@@ -1,5 +1,8 @@
 Apache Kafka
-Copyright 2015 The Apache Software Foundation.
+Copyright 2016 The Apache Software Foundation.
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This distribution has a binary dependency on jersy, which is available under the CDDL
```
Comment (ijuma) [inline NOTICE]: Same typo here.

**Comment by guozhangwang** [comment]
@junrao Do wee need the BSD license for RocksDB dependency as well?

**Comment by junrao** [comment]
No, BSD is covered. See http://www.apache.org/legal/resolved.html#category-a

**Comment by guozhangwang** [comment]
Got it, thanks.

**Comment by ijuma** [comment]
LGTM

**Comment by junrao** [comment]
Include the CDDL license directly as suggested by https://mail-archives.apache.org/mod_mbox/www-legal-discuss/201603.mbox/%3CCALGG8z1pm22YTQpoP0_n4xSomXHR6P-2gn4ie9VtLDLvhA2-ZA@mail.gmail.com%3E

**Comment by gwenshap** [comment]
LGTM

**Pull request closed without merging**